includes/class-wsu-syndicate-shortcode-people.php

```diff
@@ -329,6 +338,11 @@ private function generate_item_html( $person, $type, $atts ) {
 			$photo = $person->profile_photo;
 		}
 
+		// If all else fails, let's set a default placeholder image
+		if ( empty($photo) ) {
```

Comment:
Let's only add the placeholder image for the card type with an if ( 'card' == $type ) check.